sdk/agrifood/Azure.Verticals.AgriFood.Farming/src/autorest.md

```diff
@@ -4,10 +4,13 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ```yaml
 title: FarmBeats
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/683e3f4849ee1d84629d0d0fa17789e80a9cee08/specification/agfood/data-plane/Microsoft.AgFoodPlatform/preview/2021-03-31-preview/agfood.json
+input-file:
+- C:\Users\bhkansag\bhargav-kansagara\azure-rest-api-specs-pr\specification\agrifood\data-plane\Microsoft.AgFoodPlatform\preview\2021-07-31-preview\agfood.json
```

Comment:
This will need to be changed to point to a spec in the public repo.

Reply:
Updated